test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -465,7 +465,7 @@ public void EntryPointCreateEnsemble()
                      MetadataUtils.Const.ScoreValueKind.Score)
                     ).Transform(individualScores[i]);
 
-                individualScores[i] = ColumnSelectingTransformer.CreateDrop(Env, individualScores[i], MetadataUtils.Const.ScoreValueKind.Score);
+                individualScores[i] = new ColumnSelectingTransformer(Env, null, new[] { MetadataUtils.Const.ScoreValueKind.Score }).Transform(individualScores[i]);
```

Comment:
>null [](start = 74, length = 4)

this looks funny, but i guess the ColumnSelectingTransformer is internal.

Reply:
I preferred the CreateDrop/CreateKeep and make this constructor private to avoid confusion. 

---
In reply to: [254400947](https://github.com/dotnet/machinelearning/pull/2417#discussion_r254400947) [](ancestors = 254400947)